tests/CMakeLists.txt

```diff
@@ -13,16 +13,27 @@ include("${PROJECT_SOURCE_DIR}/../cmake/get_cpp.cmake")
 
 function(add_cmake_test_test)
```

Comment:
We're presumably going to need similar functionality downstream in CMakePP and CMaize as well so I recommend making this into a function which takes a directory as input and registers each *.cmake* file in that directory (and its subdirectories) as unit tests with CTest.